bml/lib/data_summary.py

```diff
@@ -59,37 +63,42 @@ def print_run_details(config, es_backend, uuid, update):
         test_name = test_type['test']
         data = []
         test_run = None
+        check_outcome = 0  # just to see if it passses all conditions
```

Comment:
so closer to the return codes concept, do you know if it's required now that errors are handled early again?